spec/spec_helper.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require 'dry/monads'
```

Comment:
@jeremyf this dry-monads is now a dev dependency of bulkrax with this change. but I dont see where you are actually using it? can this come out or do we need to add it to the gemspec?

Reply:
Without the `require 'dry/monads'` the specs failed to load.  This was the quickest adjustment.

Looking at it now, my assumption is that `dry/monads` is a development dependency in that we are creating transactions and merging those transactions.

So the shorter solution would be to move this to a development dependency.